meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0+20170516.bb

```diff
@@ -0,0 +1,293 @@
+# Recipe created by recipetool
+# This is the basis of a recipe and may need further editing in order to be fully functional.
+# (Feel free to remove these comments when editing.)
+
+SUMMARY = "zebkit"
+HOMEPAGE = "https://github.com/barmalei/zebra"
+# WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
```

Comment:
@akilawickey You need to edit the recipe provided by recipetool.
Comments like these should not be in the final proposal.  
It seems like it has a crazy amount of license information.  Maybe it is necessary to give the full picture, but we will need to review it.